docs/ref/class-based-views/generic-display.txt

```diff
@@ -57,7 +57,7 @@ many projects they are typically the most commonly used views.
 
         from django.urls import path
 
-        from article.views import ArticleDetailView
+        from articles.views import ArticleDetailView
```

Comment:
Shouldn't it be `myapp.views` or simply `.views`? `articles` is coherent with the `articles.models` import in `myapp.views`.

It seems like there are other inconsistencies here.

Reply:
I think you could change `Example myapp/views.py` to `Example views.py` and be consistent with what is in base.txt